### ISSUE TITLE
Board-specific CSS gets used for some mod pages

### ISSUE THREAD

**Issue by hawkoon** [vichan-devel/vichan#368]
TL;DR: Board-specific CSS is leaking into some mod pages.

Extremely weird bug here. I have three boards in total on my Vichan instance. Two of those are using their own config.php (in the folder of their respective board uri, as per instructions). Now there's something very weird happening. It seems the board-specific CSS of the 3rd (alphabetically last) board gets cross-used on some mod.php pages. For instance my Rebuild page uses the normal, global (or 'default') stylesheet, yet when I click 'Rebuild' and see the 'Rebuilt' page suddenly the page uses the board-specific CSS of my third board. That CSS is also used on the View IP page, for example. But as far as I know all the other mod pages use the normal default CSS just fine as expected. I have checked the config.php of the board in question and there is nothing weird there, only 3 lines and no coding errors. See also the screenshots attached to clarify my examples.

![default theme](https://user-images.githubusercontent.com/54553501/73386416-18cd6980-42cf-11ea-889b-c0c7cf3b8792.png)

![rebuilt](https://user-images.githubusercontent.com/54553501/73386599-677b0380-42cf-11ea-85bb-9f5c65b04162.png)

![mod_page](https://user-images.githubusercontent.com/54553501/73386430-1e2ab400-42cf-11ea-8b23-43f90913fde4.png)

**Comment by john-clever** [comment]
Are you using 7.4? This happened with me when the last(or first, I don't remember) report or appeal was from a board with its css. Did you check it out? Maybe the last board being rebuild is changing the css as well, I don't think that's a bug, but as a temp fix you can put a config default stylesheets right before rebuild in inc/mod/pages.php

**Comment by hawkoon** [comment]
> 
> 
> Are you using 7.4? This happened with me when the last(or first, I don't remember) report or appeal was from a board with its css. Did you check it out? Maybe the last board being rebuild is changing the css as well, I don't think that's a bug, but as a temp fix you can put a config default stylesheets right before rebuild in inc/mod/pages.php

No I was using PHP 7.2.
>you can put a config default stylesheets right before rebuild in inc/mod/pages.php

What do you mean by that?

**Comment by ctrlcctrlv** [comment]
I think I fixed this bug many years ago as 8chan's admin, since it was a serious security problem on 8chan.

On Vichan it's not a security problem since users won't be specifying the stylesheets, so...this is not a big deal IMHO
